.github/workflows/example.yml

```diff
@@ -64,16 +62,19 @@ jobs:
       # 1. Installs everything in package-lock.json.
       # 2. Checks that package.json and package-lock.json are in sync.
       # That’s why the cache depends on both package-lock.json and package.json.
-      - name: npm ci
-        if: steps.cache-node_modules.outputs.cache-hit != 'true'
+      # `--no-audit` is used because nobody looks at audit reports in CI –
+      # it’s just a waste of time.
```

Comment:
This is clever, I'll be using this trick in the future. Thanks for the inspiration!